pandas-stubs/io/json/_json.pyi

```diff
@@ -11,194 +9,89 @@ from pandas.core.series import Series
 
 from pandas._typing import (
     CompressionOptions,
-    FilePathOrBuffer,
-    JSONSerializable,
+    DtypeArg,
+    FilePath,
+    HashableT,
+    ReadBuffer,
+    StorageOptions,
 )
 
-loads = ...
-dumps = ...
-
-def to_json(
-    path_or_buf,
-    obj,
-    orient: str | None = ...,
-    date_format: str = ...,
-    double_precision: int = ...,
-    force_ascii: bool = ...,
-    date_unit: str = ...,
-    default_handler: Callable[[Any], JSONSerializable] | None = ...,
-    lines: bool = ...,
-    compression: str | None = ...,
-    index: bool = ...,
-    indent: int = ...,
-): ...
-
-class Writer:
-    obj = ...
-    orient = ...
-    date_format = ...
-    double_precision = ...
-    ensure_ascii = ...
-    date_unit = ...
-    default_handler = ...
-    index = ...
-    indent = ...
-    is_copy = ...
-    def __init__(
-        self,
-        obj,
-        orient: str | None,
-        date_format: str,
-        double_precision: int,
-        ensure_ascii: bool,
-        date_unit: str,
-        index: bool,
-        default_handler: Callable[[Any], JSONSerializable] | None = ...,
-        indent: int = ...,
-    ) -> None: ...
-    def write(self): ...
-
-class SeriesWriter(Writer): ...
-class FrameWriter(Writer): ...
-
-class JSONTableWriter(FrameWriter):
-    schema = ...
-    obj = ...
-    date_format = ...
-    orient = ...
-    index = ...
-    def __init__(
-        self,
-        obj,
-        orient: str | None,
-        date_format: str,
-        double_precision: int,
-        ensure_ascii: bool,
-        date_unit: str,
-        index: bool,
-        default_handler: Callable[[Any], JSONSerializable] | None = ...,
-        indent: int = ...,
-    ): ...
-
 @overload
 def read_json(
-    path: FilePathOrBuffer,
-    orient: str | None = ...,
-    dtype=...,
-    convert_axes=...,
-    convert_dates: bool = ...,
+    path_or_buf: FilePath | ReadBuffer[str] | ReadBuffer[bytes],
+    *,
+    orient: Literal["split", "records", "index", "columns", "values", "table"]
+    | None = ...,
```

Comment:
Don't see that `None` is an allowable argument here in the docs.  Which probably means the docs need to be updated.  Can you create an issue in pandas repo for that?

Reply:
`None` is special since it handles the different defaults automatically.

Reply:
Issue opened.